lib/services/ios-debug-service.ts

```diff
@@ -72,14 +74,15 @@ class IOSDebugService implements IDebugService {
         }).future<void>()();
     }
 
-    private emulatorDebugBrk(): IFuture<void> {
+    private emulatorDebugBrk(shouldBreak? : boolean): IFuture<void> {
```

Comment:
>we prefer using objects and pass a property of them

It is not a matter of what is preferred but what works. Different teams have different cultures. Thanks for suggestion but please try to focus on arguments that both teams consider as constructive.